runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -616,16 +610,21 @@ pub fn run(mut config: Config, only_compile: bool) -> RuntimeConfig {
         );
     }
 
-    get_runtime_config(&m, &config)
+    let contract_compile_costs = compute_compile_cost_vm(config.metric, config.vm_kind, false);
+    let wasm_instr_cost = ratio_to_gas(config.metric, cost_per_op(config.metric, &CODE_1M));
+
+    let mut cost_table = measurements_to_costs(m, contract_compile_costs, wasm_instr_cost);
+
+    if let Some(gas) = cost_table.get(Cost::StorageReadBase) {
+        // TODO: Actually compute it once our storage is complete.
```

Comment:
Not sure how touching cost can be several time less than read operation.

Reply:
Yeah, the correctness here is questionable, especially given that we didn't enable IO costs. But the logic seems fine to me -- a single storage_read operation needes to touch several trie nodes.

Reply:
So what's the meaning of "touching" then? Anyway, not to relevant for this PR.

Reply:
My understanding (guess, really) is that `read` operation reads a key from a trie search tree. To read a key from search tree, you need to traverse several interior nodes. Touch describes reading a single node from rocks.